Makefile

```diff
@@ -121,8 +121,10 @@ install-test:
 container-test:
 	docker run --rm -ti -v ${PWD}:/payload leapp-tests
 
-test:
-	py.test --flake8 --cov-report term-missing --cov=leapp tests/scripts
-	py.test --flake8 leapp
+test:   lint
+	py.test --cov-report term-missing --cov=leapp tests/scripts
 
-.PHONY: clean copr_build install install-deps install-test srpm test
+lint:
+	py.test --cache-clear --flake8 --pylint -m pylint leapp tests/scripts/*.py
```

Comment:
```suggestion
	py.test --cache-clear --flake8 --pylint -m flake8,pylint leapp tests/scripts/*.py
```

Reply:
Having 2 modules `-m pylint,flake8` causes an unpleasant side-effect of running unit tests together with linter. I'd honestly prefer to have only linter checks under lint target.